exporter/collector/googlemanagedprometheus/naming.go

```diff
@@ -15,25 +15,34 @@
 package googlemanagedprometheus
 
 import (
+	"strings"
+
 	"go.opentelemetry.io/collector/model/pdata"
 	"go.opentelemetry.io/collector/pdata/pmetric"
 )
 
 func GetMetricName(baseName string, metric pdata.Metric) string {
-	return baseName + gmpMetricSuffix(metric)
-}
-
-func gmpMetricSuffix(metric pdata.Metric) string {
 	switch metric.DataType() {
 	case pmetric.MetricDataTypeSum:
-		return "/counter"
+		return baseName + "/counter"
 	case pmetric.MetricDataTypeGauge:
-		return "/gauge"
+		return baseName + "/gauge"
 	case pmetric.MetricDataTypeSummary:
-		return "/summary"
+		// summaries are sent as the following series:
+		// * Sum: prometheus.googleapis.com/<baseName>_sum/summary:counter
+		// * Count: prometheus.googleapis.com/<baseName>_count/summary
+		// * Quantiles: prometheus.googleapis.com/<baseName>/summary
+		if strings.HasSuffix(baseName, "_sum") {
```

Comment:
Just to confirm, `_sum` and `_count` series are sent as cumulatives and quantile series are sent as gauges?

Reply:
Yes, that is correct.  In the gmp integration test, here is the [_sum series for a summary](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/342/files#diff-92007daaaf28c12906bcd7ab2014ab132aaca919b99bbcfe6b1cca41c0510afcR227), here is the [_count series](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/342/files#diff-92007daaaf28c12906bcd7ab2014ab132aaca919b99bbcfe6b1cca41c0510afcR258), and here is a [quantile series](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/342/files#diff-92007daaaf28c12906bcd7ab2014ab132aaca919b99bbcfe6b1cca41c0510afcR290).

The logic is the same for the standard (non-gmp) exporter.